types/logger/logger.go

```diff
@@ -132,7 +132,7 @@ func RateLimitedFn(logf Logf, f time.Duration, burst int, maxCache int) Logf {
 			logf(format, args...)
 		case warn:
 			// For the warning, log the specific format string
-			logf("[RATE LIMITED] format string \"%s\"", format)
+			logf("[RATE LIMITED] %s (format string \"%s\")", fmt.Sprintf(format, args...), format)
```

Comment:
I think we should reverse the order here: First print the format string, then the example. The rationale is that it is the format string that is rate limited, so that's the most important thing and the thing that should be next to "RATE LIMITED".